src/components/People/Portfolio.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components'
+import { Headline } from './Profile'
+import Video from '../People/img/Profile_Video.mp4'
+import Portfolioimg from '../People/img/Portfolio.png'
+import PortfolioDatei from '../People/img/Portfolio-komprimiert Kopie.pdf'
+
+function Portfolio() {
+  return (
+    <Wrapper>
+      <Headline>Da wird die Milch sauer!</Headline>
+      <a href={Video}>
+        <Image
+          src={Portfolioimg}
+          alt="Werbung für Andros, eine vegane Joghurtalternative"
+        />
+      </a>
+      <div>
+        <a href={PortfolioDatei}>
```

Comment:
Here is the same problem with the button in a anchor element I wrote about above.